### ISSUE TITLE
trunk version trails behind tagged version

### ISSUE THREAD

**Issue by arth1** [koalaman/shellcheck#932]
#### For bugs
- [X] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

$ basename $(pwd); svn up && grep '^Version:' ShellCheck.cabal

```

#### Here's what shellcheck currently says:

trunk
Updating '.':
At revision 874.
Version:          0.4.5

#### Here's what I wanted or expected to see:

trunk
Updating '.':
At revision 874.
Version:          0.4.7

This prevents cabal install from the trunk directory if you already have installed the official 0.4.6 release through cabal update, because the trunk version is lower.

**Comment by andersk** [comment]
Are you accessing GitHub with a Subversion client through the [GitHub Subversion bridge](https://help.github.com/articles/support-for-subversion-clients/)? Most GitHub project maintainers will not be expecting that, and you be sure to should point it explicitly out in bug reports. I recommend [switching to a real Git client](https://help.github.com/articles/set-up-git/), though.

That said, I cannot reproduce this problem. The commit that GitHub calls r874 is b1486ec1e96345b00600ef7d2b17dc3d93a539d4, which has `Version: 0.4.6`.

If you are accessing the shellcheck repository through Subversion somewhere *other* than the GitHub Subversion bridge, that’s even more unexpected, and you should probably report a bug to wherever you found this strange mirror. (But really, you want to just use Git.)

**Comment by koalaman** [comment]
I didn't even know GitHub had a svn bridge :O 

I'm also unable to reproduce it after following that doc link. Here's what I get:

```
$ svn co --depth empty https://github.com/koalaman/shellcheck
Checked out revision 894.

$ cd shellcheck/

$ svn up trunk
Updating 'trunk':
A    trunk
A    trunk/.ghci
A    trunk/.github
A    trunk/.github/ISSUE_TEMPLATE.md
[...]

$ cd trunk

$  basename $(pwd); svn up && grep '^Version:' ShellCheck.cabal
trunk
Updating '.':
At revision 894.
Version:          0.4.6
```
